src/pages/[id].page.jsx

```diff
@@ -41,17 +39,40 @@ export default function Details() {
   );
 }
 
-export const getStaticPaths = async () => ({
-  paths: [
-    '/accordion',
-    '/dialog',
-    '/disclosure',
-    '/tabs',
-  ],
-  fallback: 'blocking', // indicates the type of fallback
-});
+export async function getStaticPaths() {
+  return {
+    paths: [
+      {
+        params: { id: 'accordion' },
+      },
+      {
+        params: { id: 'dialog' },
+      },
+      {
+        params: { id: 'disclosure' },
+      },
+      {
+        params: { id: 'tabs' },
+      },
+    ],
+    fallback: 'blocking', // indicates the type of fallback
+  };
+}
 
 // pre-render this page on each request using data in props.
-export const getStaticProps = async () => ({
-  props: {},
-});
+export async function getStaticProps({ params }) {
+  try {
+    const { id } = params;
+    // run the airtable function to get the data and save to the details variable.
+    const details = await getComponentDetails(id);
+    // pass that details variable and its data into props to use elsewhere
+    return {
+      props: {
+        componentName: details['Component Name'],
+      },
+    };
+  } catch {
+    // throw new Error('No records matched the component name.');
```

Comment:
I call these types of comments "zombie code" since they can come back from the dead and wreak havoc. Let's get rid of this 🧟

Reply:
https://github.com/sparkbox/accessible-components/pull/34/commits/0a92e87571835e5c17d53cc5a0832efd04b5d3c1

Better!